modules/knative-eventing-CR-system-deployments.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: REFERENCE
+[id="knative-eventing-CR-system-deployments_{context}"]
+= Overriding Knative Eventing system deployment configurations
+
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeEventing` custom resource (CR). Currently, overriding default configuration settings is supported for the `eventing-controller`, `eventing-webhook`, and `imc-controller` fields.
+
+In the following example, the `KnativeEventing` CR overrides the `eventing-controller` deployment to request 0.3 CPU and 100MB of RAM for the `eventing-controller` container, and sets hard limits of 1 CPU and 250MB RAM:
+
+.KnativeEventing CR example
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  deployments:
+  - name: eventing-controller
+    resources:
+    - container: eventing-controller
+      requests:
+        cpu: 300m
+        memory: 100M
+      limits:
+        cpu: 1000m
+        memory: 250M
+----
+
+[NOTE]
+====
+The `KnativeEventing` CR label and annotation settings override the deployment's labels and annotations for both the deployment itself and the resulting pods.
```

Comment:
This note mentions "labels" and "annotations", while the example above does not use any. (or is it anywhere described that those can also be used for KnativeEventing deployment overrides)

(compared to KnativeServing example http://file.rdu.redhat.com/abrennan/260722/SRVKE-1239/serverless/admin_guide/serverless-configuration.html#knative-serving-CR-system-deployments_serverless-configuration )

I think we should add also the annotations, labels, replicas and the nodeSelector as we do in the KnativeServing (AFAIK we support them too for KnativeEventing , right @pierDipi ?  (I can see that they do apply in 1.24)

Reply:
(one potential caveat though, `replicas` override won't work for deployments that use a HorizontalPodAutoscaler, so it won't work for `eventing-webhook`)